core/input/keyboard.js

```diff
@@ -126,6 +126,15 @@ export default class Keyboard {
             return;
         }
 
+        // If a key is pressed while meta is held down, the keyup will
+        // never be sent in Chrome (bug #108404) and possibly others
+        if (e.metaKey && keysym !== KeyTable.XK_Meta_L && keysym !== KeyTable.XK_Meta_R) {
```

Comment:
This is a macOS specific bug, so we want to avoid punishing other platforms.

Reply:
Don't know if it could also happen on IOS keyboard. But since the rewrite for both Super-key modifiers is guarded by `browser.isMac() || browser.isIOS()` I'll add it there.